webchat/components/header/ecis-header.component.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    'use strict';
+
+    angular.module('webchat').component('ecisHeader', {
+        templateUrl: "app/components/header/ecis-header.html",
+        controller: headerController,
+        controllerAs: "headerCtrl",
+        bindings: {
+            className: "@",
+            style: "@",
+            ariaLabel: "@",
+            mdColors: "@",
+            user: "<"
+        },
+    });
+
+    function headerController ($mdSidenav) {
```

Comment:
use inline array annotation here and in all dependency injectors
https://docs.angularjs.org/guide/di#inline-array-annotation